[core]/es_extended/client/main.lua

```diff
@@ -713,3 +713,65 @@ for i=1, #DoNotUse do
 		print("[^1ERROR^7] YOU ARE USING A RESOURCE THAT WILL BREAK ^1ESX^7, PLEASE REMOVE ^5"..DoNotUse[i].."^7")
 	end
 end
+
+-- Marker Interaction System
+local DrawingTextUI = false
+CreateThread(function()
+	while not ESX.PlayerLoaded do
+		Wait(0)
+	end
+	while true do
+		local Sleep = 1000
+		local ShowTextUI = false
+		local PlayerCoords = GetEntityCoords(ESX.PlayerData.ped)
+		if ESX.Table.SizeOf(Core.Markers) < 1 then
```

Comment:
```suggestion
    if not next(Core.Markers) then
```
for better performance 😁